### PR TITLE
Fix error reporting in docker mode

### PR DESCRIPTION
**Pull request by FriggaHel**
Fix error reporting in docker mode.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -154,8 +154,29 @@ function renameAsset (specFile, oldFilePath, resultsFolder) {
...
 
+// Store file containing job-details url.
+// Path is similar to com.saucelabs.job-info LABEL in Dockerfile.
+const OUTPUT_FILE_PATH = '/tmp/output.json';
+
+function exportValueToSaucectl (payload) {
+  fs.writeFileSync(OUTPUT_FILE_PATH, JSON.stringify(payload));
+}
+
+function updateExportedValueToSaucectl (data) {
```
Comment (alexplischke) [inline src/utils.js:165]: Hm, do you think there's a better file than `utils.js` to store such methods?
I was hoping we could eventually get rid of the `utils.js` completely, because they tend to collect a lot of methods over time that are unrelated to each other :D

Comment (FriggaHel) [reply src/utils.js]: This has been splitted to `saucectl-exporter.js` :)

**Review by alexplischke — APPROVED** [review]

**Comment by alexplischke** [comment]
PS: Might be a good opportunity to start moving stuff like that to the shared [sauce runner utils](https://github.com/saucelabs/sauce-runner-utils/)! :)

**Review by alexplischke — APPROVED** [review]
